.github/actions/setup-playwright/requirements.txt

```diff
@@ -0,0 +1 @@
+pytest-playwright
```

Comment:
Please pin to a version to manage the hit or not of the cache. =)

Reply:
This version cannot be pin for two reason
1. python version is variable so we cannot ensure that the pinned version will be compatible with all the python versions
2. Pinned versions are just useful when there are conflicts and pip should install the last available version compatible with the current version 

The cache is handled by the python version if that changes  the cache shouldn't return anything